packages/sui-react-web-vitals/src/index.js

```diff
@@ -0,0 +1,68 @@
+import {useContext, useEffect, useRef} from 'react'
+
+import {onCLS, onFID, onLCP, onTTFB} from 'web-vitals'
```

Comment:
it could be useful to be able to customize which metrics we want to track via props / config... what do you think?

Reply:
Agree

Reply:
Done